gui/gui/inc/TGDimension.h

```diff
@@ -40,6 +40,8 @@ class TGDimension {
       { return TGDimension(fWidth - b.fWidth, fHeight - b.fHeight); }
    TGDimension operator+(const TGDimension &b) const
       { return TGDimension(fWidth + b.fWidth, fHeight + b.fHeight); }
+   TGDimension &operator=(const TGDimension &d)
```

Comment:
If I were you I'd rather remove the copy constructor, and rely on the compiler generated one?

And while we're at it, it seems that nobody is inheriting from `TGDimension`, is there a reason why it has a virtual destructor? It likely shouldn't - consider just removing the destructor, too!

And I don't think we need `ClassDef` here, either!

Same for the other occurrences of this change.

Reply:
OK, then I'll review those classes (since I'm not the author, I just added the missing piece of code...)